frontend/src/seqvars/queries/seqvarQueryPresetsFrequency.ts

```
@@ -0,0 +1,210 @@
+      // server-maintained fields such as `date_created` and `date_modified` otherwise.
+      const newValue =
+        previousValue === undefined && data.body === undefined
+          ? undefined
+          : deepmerge(previousValue, data.body as SeqvarsQueryPresetsFrequency)
+      queryClient.setQueryData(queryKey, newValue)
+      // Return a context with the previous and new data.
+      return { previousValue, newValue }
+    },
+    // If the mutation fails, rollback with context we returned above.
+    onError: (_err, variables, context) => {
+      if (!!context?.previousValue) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)